addons/web_editor/static/src/less/web_editor.variables.less

```diff
@@ -155,11 +193,9 @@
 
         &.btn-primary {
             .button-variant(white, @odoo-brand-optional, darken(@odoo-brand-optional,5%));
-            text-transform: uppercase;
```

Comment:
Sorry, I forgot to reply!
So, uppercase and capitalized text play an huge role to guide the user through the UI.
The usage should be set progressively, following an importance criteria.
If we design **everything** as important (uppercase), nothing will be perceived as important anymore.

You'll find tons of references about that: 
- https://msdn.microsoft.com/en-us/library/dn742478.aspx
- https://medium.com/@jsaito/making-a-case-for-letter-case-19d09f653c98
- https://code.vmware.com/web/standards/-/capitalization-of-ui-elements
- http://uxmovement.com/content/all-caps-hard-for-users-to-read/
- [...]

Now, I know that in the backend we use to uppercase any btns (don't know why),  and I assume you want to keep consistency.. but since for the fronted we introduced the preservation mixins, I don't why "forcing the preservation" of something wrong 😕